eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -253,7 +253,7 @@ jobs:
 
       - task: Maven@3
         displayName: 'Start Jetty'
-        condition: ne('${{ parameters.SDKType }}', 'client')
+        condition: and(succeeded(), ne(variables['SdkType'], 'client'))
```

Comment:
@mitchdenny why was this changed from parameters back to variables? Isn't variables going to try and pull from the pipeline variable SDKType? Using the parameters should, in theory, guarantee that this only runs for the correct track.

Reply:
My bad ... I'll fix it up now. I was getting shown some interesting conflict resolutions when I did my rebase.